### PR TITLE
Internal: Editor - Rearrange promotion dialog [ED-7697]

### PR DESCRIPTION
**Pull request by Nevoss**

### CODE REVIEW

**Review by matipojo — CHANGES_REQUESTED** [review]
🔥🔥

File: assets/dev/js/editor/components/dynamic-tags/control-behavior.js [hunk 1]
```diff
@@ -197,15 +197,28 @@ module.exports = Marionette.Behavior.extend( {
...
-		const message = __( 'Create more personalized and dynamic sites by populating data from various sources with dozens of dynamic tags to choose from.', 'elementor' );
-
-		elementor.promotion.showDialog( {
-			headerMessage: __( 'Dynamic Content', 'elementor' ),
-			message,
-			top: '-10',
-			element: this.ui.dynamicSwitcher,
-			actionURL: elementor.config.dynamicPromotionURL.replace( '%s', this.view.model.get( 'name' ) ),
-		} );
+			const isProExistNotConnected = ! ! ( elementor.helpers.hasPro() && elementorProEditorConfig.urls.connect ),
```
[the remark this answers is not in the document]
Comment (Nevoss) [reply assets/dev/js/editor/components/dynamic-tags/control-behavior.js]: Done

File: assets/dev/js/editor/utils/helpers.js [hunk 2]
```diff
@@ -658,6 +658,10 @@ module.exports = {
 		return !! window.elementorPro;
 	},
 
+	hasProAndNotConnected() {
+		return elementor.helpers.hasPro() && elementorProEditorConfig.urls.connect;
```
Comment (matipojo) [inline assets/dev/js/editor/utils/helpers.js:662]:
```suggestion
		return this.hasPro() && elementorProEditorConfig.urls.connect;
```

**Review by matipojo — CHANGES_REQUESTED** [review]

File: assets/dev/js/editor/utils/helpers.js [hunk 3]
```diff
@@ -658,6 +658,10 @@ module.exports = {
 		return !! window.elementorPro;
 	},
 
+	hasProAndNotConnected() {
+		return this.hasPro() && elementorProEditorConfig.urls.connect;
```
Comment (matipojo) [inline assets/dev/js/editor/utils/helpers.js:662]:
```suggestion
		return elementor.helpers.hasPro() && elementorProEditorConfig.urls.connect;
```

**Pull request merged**